apps/wakucanary/wakucanary.nim

```diff
@@ -24,6 +26,7 @@ const ProtocolsTable = {
 }.toTable
 
 const WebSocketPortOffset = 1000
+const CertsDirectory = "./apps/wakucanary/certs"
```

Comment:
I would either make this configurable, or used some more general path like `./certs` or `./self-signed-certs` - WDYT?

Reply:
Yes! Wasn't sure what was the best approach. Didn't want to make it configurable so we don't require another CLI flag (the whole idea is for this to happen under the hood/on its own). Also wanted to keep it inside the `wakucanary` directory so it doesn't mix with other stuff.

So I assumed that the instructions are being followed literally and it would be run from the root of the repo.

But makes sense, maybe it's better to store the certs in the directory where the executable is run, and avoid errors if it's run from a different place. Fixing it now :)